<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vendedor logado</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(129, 129, 129);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botao_sair">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 132, 53);
color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>Sair?</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_logado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>adimin_Luca est√° logado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="enviarCad">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>182</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 132, 53);
color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>Verificar produtos cadastrados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enviarCad_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>182</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(76, 132, 53);
color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>Cadastrar novo produto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Perfil64.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
